srcpkgs/spotifyd/template

```diff
@@ -1,25 +1,36 @@
 # Template file for 'spotifyd'
 pkgname=spotifyd
 version=0.2.24
-revision=1
+revision=2
 build_style=cargo
+configure_args="--no-default-features"
 hostmakedepends="pkg-config"
-makedepends="alsa-lib-devel libressl-devel pulseaudio-devel dbus-devel"
+makedepends="libressl-devel $(vopt_if alsa alsa-lib-devel)
+$(vopt_if pulseaudio pulseaudio-devel) $(vopt_if dbus dbus-devel)"
 short_desc="Spotify client running as a UNIX daemon"
 maintainer="Andrew Benson <abenson+void@gmail.com>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/Spotifyd/spotifyd"
 distfiles="${homepage}/archive/v${version}.tar.gz"
 checksum=d3763f4647217a8f98ee938b50e141d67a5f3d33e9378894fde2a92c9845ef80
 
+build_options="alsa pulseaudio dbus"
+build_options_default="alsa pulseaudio dbus"
+desc_option_alsa="Enable ALSA backend"
```

Comment:
These options have default description in common/options.description

Reply:
Do you think we could add this as a check to xlint or would that be too complicated?